### PR TITLE
[v1.1][ISSUE-188] Refine Distiller close file and UT

### PR DESCRIPTION
**Pull request by XinyaoWa**
- Due to code security scan, add close file to after read files in save logits function 

- Add save logits function check in UT

### CODE REVIEW

**Review by xuechendi — APPROVED** [review]
LGTM

**Comment by xuechendi** [comment]
@XinyaoWa , I found github hosted runner seems has some issue today, I restarted the first action but it kept pending for github to assign runner. So you don't need to expect first two UT to complete.  Need to test UnitTest locally. 
image

**Comment by XinyaoWa** [comment]
> @XinyaoWa , I found github hosted runner seems has some issue today, I restarted the first action but it kept pending for github to assign runner. So you don't need to expect first two UT to complete. Need to test UnitTest locally. image

 I have tested the UT locally and they passed. So should I wait for fixing, or directly merge?
